### DISCUSSION TITLE
Forcing operations to be executed in succession?

### DISCUSSION THREAD

**Discussion by MathiasMantai** [frePPLe/frepple#607] [Q&A]
Hi,
I wanted to ask if it's possible to force operations to be executed in succession. I have read about Operation dependencies but the description of the fields doesn't sound like what I'm after. Mainly taking about the field safety_leadtime.

Ty for all replies

**Comment by jdetaeye** [comment]
See https://github.com/frePPLe/frepple/discussions/600

**Comment by zipus** [comment]
Also see #579
